src/org/loklak/LoklakInstallation.java

```diff
@@ -130,7 +130,10 @@ public static void main(String[] args) throws Exception {
         if(env.containsKey("PORTSSL")) {
             httpsPort = Integer.parseInt(env.get("PORTSSL"));
         }
-        
+
+        if (args.length > 0) {
+            httpPort = Integer.valueOf(args[0]);
```

Comment:
@djmgit see this -> http://stackoverflow.com/questions/7355024/integer-valueof-vs-integer-parseint